_posts/2019-05-01-#15557.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: pr
 title: "Enhance bumpfee to include inputs when targeting a feerate"
-components: [wallet]
+components: ["wallet"]
```

Comment:
https://github.com/bitcoin/bitcoin/pull/15557 seems to be tagged "tx fees and policy"

Reply:
No, this is definitely a wallet PR